src/common/utility.ts

```diff
@@ -424,3 +424,8 @@ export const handleRowRedirect = (residence: string, current_client_country: str
 export const isLocalhost = () => !!(isBrowser() && process.env.NODE_ENV === 'development')
 
 export const isTestlink = () => !!(isBrowser() && window.location.hostname.includes('binary.sx'))
+
+export const getDateFromToday = (num_of_days: number) => {
```

Comment:
@mitra-deriv Try to write unit test for the newly added utility function.

Reply:
This change was removed from master in resolving conflicts I kept it by mistake, thanks for pointing it out, Removed it as it's not part of my changes.